dev/release/02-source.sh

```diff
@@ -17,6 +17,10 @@
 # specific language governing permissions and limitations
 # under the License.
 #
+: ${PREPARE_DEFAULT:=1}
```

Comment:
Could you put all codes after `set -e` to check all codes by `set -e`?

Reply:
`SOURCE_` prefix is better.

Reply:
I've moved all codes after `set -e`.